### PR TITLE
new/conan

### LINKED ISSUE #15: Setup Conan
The main reason to switch over to conan is for cross-compilation ease. The circleci test is not passing because the build is only for Windows, but it is targeting Linux on the container. 

Reminder: Add SDL to conan

### PR DESCRIPTION
**Pull request by richardllara05**
new/conan Changes have been completed. Closes #15.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 target_include_directories(
 	${PROJECT_NAME}
 		PRIVATE ${INC}
-		PRIVATE ${SDL2_INCLUDE_DIRS}
 )
 
-
-target_link_libraries(${PROJECT_NAME} PRIVATE ${SDL2_LIBRARIES})
\ No newline at end of file
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
```
Comment (jaburialeh) [inline CMakeLists.txt:17]: Can you add more than 1 version of C++, Iike C++ 11 too? Just curious

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+project(chip8)
+
 file(GLOB SRC src/*.cc)
 file(GLOB INC inc/*.h)
 
-include(deps/CMakeLists.txt)
-project(chip8)
-
-find_package(sdl2)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```
Comment (jaburialeh) [inline CMakeLists.txt:8]: What does this include add? Is it a file inside the ${CMAKE_BINARY_DIR}?

**Review by jaburialeh — APPROVED** [review]
Looks awesome, great job making the conan changes to add sdl🔥

File: CMakeLists.txt (hunk 2 above)
Comment (richardllara05) [reply CMakeLists.txt:8]: Yes, include is used to add other files with cmake syntax. Typically *.cmake files.

File: CMakeLists.txt (hunk 1 above)
Comment (richardllara05) [reply CMakeLists.txt:17]: Not that I know of, but you can use a combination of conditionals with an argument passed to cmake to change versions.

**Review by Willygilly — APPROVED** [review]

**Pull request merged**